gateway/helm/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: gateway
 description: A Helm chart for the Grafbase Gateway
 type: application
-version: 0.1.0
+version: 0.2.0
```

Comment:
I have a question about the release process.
The helm chart code did not change in this PR.
- it's a new release because we have new code between now and the previous release
- or, we bump the helm chart version because of the new application version

Reply:
I missed initially the version change from latest. 
Ignore my previous comment

Reply:
This is an interesting question. 
I don't think it's a requirement, and personally I don't see the benefits.
1. The meaning of the change will be opaque to the users, if they don't read the helm code.
2. In practice, this forces users automatically to follow the `latest`, which can be achieved with proper versioning of the container image
4. I would prefer for me, as a user, to have control over the version. Usually this version is the container version that I would like to use
5. if the company follows best practices with versioning, this can be achieved with containers tagged with ":0.27", "0.27.1", etc

Having said that, this definitely should be discussed and for now we should follow the existing workflows, meaning that this LGTM

Reply:
We do tag the containers. Maybe I'll write to the chart readme to change the tag to the version they want to use.

Reply:
How do popular charts like this one manage their helm releases? https://artifacthub.io/packages/helm/bitnami/postgresql